src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/softdelete/ResetService.kt

```diff
@@ -56,11 +56,11 @@ class ResetService(
   }
 
   data class ResetResponse(
-    @Schema(description = "Number of licences reset in hdc")
+    @param:Schema(description = "Number of licences reset in hdc")
```

Comment:
should schema be on `field` rather than `param`? 

Does it appear ok in swagger?